cnf-certification-test/operator/util/util.go

```diff
@@ -0,0 +1,11 @@
+package operator
+
+import "regexp"
+
+const semvarRegex = "^(0|[1-9]\\d*)\\.(0|[1-9]\\d*)\\.(0|[1-9]\\d*)(?:-((?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\\.(?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\\+([0-9a-zA-Z-]+(?:\\.[0-9a-zA-Z-]+)*))?$"
+
+func isValidSemanticVersion(version string) bool {
```

Comment:
Is it better to use a library, e.g.:
 https://github.com/Masterminds/semver

Reply:
I agree, if we can _outsource_ it to an outside library lets do it.  We already have masterminds/semver included in this repo already.

Reply:
That's cool. Thanks! I did not know about this library earlier. I will update the code and check if the unit tests pass.